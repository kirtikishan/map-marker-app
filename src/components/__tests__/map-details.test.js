import React from "react";
import { shallow } from "enzyme";
import MapDetails from "../map-details";

describe("MapDetails", () => {
  it("should render map details", () => {
    const details = {
      authority: "captain of the guard",
      name: "Oakheart",
      numGuards: 32,
      population: 3274,
      position: [89, 283],
      type: "large town",
      wealth: 299335,
    };
    const tree = shallow(<MapDetails props={details} />);
    expect(tree).toMatchSnapshot();
  });
});
