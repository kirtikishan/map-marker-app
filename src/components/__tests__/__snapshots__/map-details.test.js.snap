// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapDetails should render map details 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      NAME
    </styled.div>
    <styled.div>
      NA
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      AUTHORITY
    </styled.div>
    <styled.div>
      NA
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      NUMBER OF GUARDS
    </styled.div>
    <styled.div>
      0
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      POPULATION
    </styled.div>
    <styled.div>
      0
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      TYPE
    </styled.div>
    <styled.div>
      NA
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      WEALTH
    </styled.div>
    <styled.div>
      0
    </styled.div>
  </styled.div>
</styled.div>
`;
